hotsos/core/host_helpers/cli.py

```diff
@@ -793,7 +793,9 @@ def command_catalog(self):
             'ns_ip_addr':
                 [BinCmd('ip netns exec {namespace} ip address show'),
                  FileCmd('sos_commands/networking/'
-                         'ip_netns_exec_{namespace}_ip_address_show')],
+                         'ip_netns_exec_{namespace}_ip_address_show'),
+                 FileCmd('sos_commands/networking/namespaces/{namespace}/'
+                         'ip_netns_exec_{namespace}_ip_-d_address_show')],
```

Comment:
They also changed the filename?

Reply:
yep, it has a -d in it now

Reply:
https://github.com/sosreport/sos/blob/9591c47f2bbca5edb331aaa008cab300e557f67a/sos/report/plugins/networking.py#L207

Reply:
Cool, thanks!